### PR TITLE
Updates to Makefile

### PR DESCRIPTION
**Pull request by jfrazelle**
not everyone has godeps and all that crap on their host, namely me, i do not so
yeah.... this is better

Signed-off-by: Jessica Frazelle <acidburn@docker.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,7 @@ GO_LDFLAGS=-ldflags "-w $(CTIMEVAR)"
 GO_LDFLAGS_STATIC=-ldflags "-w $(CTIMEVAR) -extldflags -static"
 GOOSES = darwin freebsd linux
 GOARCHS = amd64
+NOTARYDIR := /go/src/github.com/docker/notary
```
Comment (diogomonica) [inline Makefile]: What if your go dir is not on /go ?

Comment (jfrazelle) [inline Makefile]: its only used in the container

File: Makefile [hunk 2]
```diff
@@ -97,12 +98,28 @@ binaries: ${PREFIX}/bin/notary-server ${PREFIX}/bin/notary ${PREFIX}/bin/notary-
...
+server-dockerfile:
+	@docker build --rm --force-rm -f notary-server-Dockerfile -t notary-server .
+
+signer-dockerfile:
+	@docker build --rm --force-rm -f notary-signer-Dockerfile -t notary-signer .
+
+docker-images: notary-dockerfile server-dockerfile signer-dockerfile
+
+shell: notary-dockerfile
+	docker run --rm -it -v $(CURDIR)/cross:$(NOTARYDIR)/cross -v $(CURDIR)/bin:$(NOTARYDIR)/bin notary bash
```
Comment (jfrazelle) [inline Makefile]: @diogomonica here

Comment (diogomonica) [inline Makefile]: got it

**Pull request merged**